nemo/collections/nlp/modules/common/transformer/transformer_modules.py

```diff
@@ -62,14 +63,23 @@ def forward(self, position_ids):
         max_pos_id = position_ids.max()
         # update positional encoding if needed
         if max_pos_id >= self._max_sequence_length:
-            self._max_sequence_length = max_pos_id + 1
+            logging.warn(
```

Comment:
based on other code this should be `logging.warning()` or `warnings.warn()`, `logging.warn()` crashes the code.